coremltools/converters/sklearn/_ridge_regression.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2017, Apple Inc. All rights reserved.
```

Comment:
It seems this file was copied from `coremltools/coremltools/converters/sklearn/_linear_regression.py`. The only difference seems be to what parameters are checked for in the `convert` method. This one checks for `Ridge`. The original checks for `LinearRegression`.

I think it would be better if we just had one copy of this code. Please modify `_linear_regression.py` to accept either `Ridge` or `LinearRegression`. 

Then I think this file could just be:
````python
# Copyright (c) 2017, Apple Inc. All rights reserved.
#
# Use of this source code is governed by a BSD-3-clause license that can be
# found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause

from . import convert, get_input_dimension

if _HAS_SKLEARN:
    import sklearn

    sklearn_class = sklearn.linear_model.Ridge
````

Please give that a try.

Reply:
@TobyRoseman I am trying to locally test the library. I am able to run `bash scripts/build.sh --python=3.7 `, but when I try `bash scripts/test.sh --python=3.7` I get the error: 


```bash
+ echo '"--test-package" is a required paramter.'
"--test-package" is a required paramter.
+ exit 1
```

If I add the `--test-package` arg (`bash scripts/test.sh --test-package --python=3.7`), I get the error:

```bash
+ unknown_option --test-package
+ echo 'Unknown option --test-package. Exiting.'
Unknown option --test-package. Exiting.
+ exit 1
```

Reply:
I wouldn't recommend running all the tests. That will take quite awhile. I would just run your new tests, and maybe also the sklearn linear regression tests.

To do this run:
``` 
source scripts/env_activate.sh
pip install -r reqs/test.pip

pytest coremltools/test/sklearn_tests/test_ridge_regression.py
pytest coremltools/test/sklearn_tests/test_linear_regression.py 
```

If the above doesn't work, just push your changes. I can kick off another CI run.

Reply:
I used the `_linearSVR` as an example to create a new `_ridge_regression`, because the `_converter_internal` requires both `convert` and `get_input_dimension` for each `sk_obj` class. It uses `_linear_regression` for conversion and checks.

Both unit tests in your comment above pass.

Please let me know if this makes sense to merge this PR. And if so, is there a timeline for merging PRs?

Reply:
Great, thanks. I like this approach a lot. That way the `convert` docstring for the ridge will make sense (it will not just be the one for linear regression).

I have kicked off another CI run:
https://gitlab.com/coremltools1/coremltools/-/pipelines/721721196

If that passes, I will merge this PR tomorrow. However please note: it will still be awhile till these changes are released.